hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/lease/LeaseManager.java

```diff
@@ -204,9 +205,8 @@ public void shutdown() {
     checkStatus();
     LOG.debug("Shutting down LeaseManager service");
     leaseMonitor.disable();
-    synchronized (monitor) {
```

Comment:
`shutdown()` should be `synchronized`.

Reply:
@szetszwo currently I have reverted this as shutdown is threadsafe. Adding synchronization needs protect all member variables as findbug, which is not required.